<ui version="4.0" >
 <author>Ulrich Van Den Hekke</author>
 <class>JavaObjectFile</class>
 <widget class="QDialog" name="JavaObjectFile" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Path of Java XML Object File :</string>
       </property>
       <property name="buddy" >
        <cstring>m_javaObjectPathEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="m_javaObjectPathEdit" />
       </item>
       <item>
        <widget class="QToolButton" name="m_javaObjectPathChange" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>List of Java Object File :</string>
         </property>
         <property name="buddy" >
          <cstring>m_javaObjectList</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_javaObjectRefreshButton" >
         <property name="text" >
          <string>&amp;Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="m_javaObjectList" >
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <item>
        <property name="text" >
         <string>JEveView</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>JEvlView</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>JEvmView</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>JEvpView</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>JTscView</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_javaObjectPathEdit</tabstop>
  <tabstop>m_javaObjectPathChange</tabstop>
  <tabstop>m_javaObjectRefreshButton</tabstop>
  <tabstop>m_javaObjectList</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>JavaObjectFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>216</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>JavaObjectFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>297</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
